broker/config/settings.yml

```diff
@@ -195,6 +195,18 @@ defaults: &defaults
     encryption:
       key: 'ABCDEFGHABCDEFGHABCDEFGHABCDEFGH'
       initialization_vector: 'ABCDEFGHABCDEFGH'
+      certificate: <%= certificate('broker.crt') %>
+      private_key: <%= certificate('broker.key') %>
+      crds:
+        "deployment.servicefabrik.io_v1alpha1_directors.yaml": base64_template('crds/deployment.servicefabrik.io_v1alpha1_directors.yaml')
+        "lock.servicefabrik.io_v1alpha1_deploymentlocks.yaml":  base64_template('crds/lock.servicefabrik.io_v1alpha1_deploymentlocks.yaml')
+        "backup.servicefabrik.io_v1alpha1_defaultrestores.yaml": base64_template('crds/backup.servicefabrik.io_v1alpha1_defaultrestores.yaml')
+        "bind.servicefabrik.io_v1alpha1_directorbinds.yaml": base64_template('crds/bind.servicefabrik.io_v1alpha1_directorbinds.yaml')
+        "bind.servicefabrik.io_v1alpha1_dockerbinds.yaml": base64_template('bind.servicefabrik.io_v1alpha1_dockerbinds.yaml')
+        "backup.servicefabrik.io_v1alpha1_defaultbackups.yaml": base64_template('crds/backup.servicefabrik.io_v1alpha1_defaultbackups.yaml')
+        "deployment.servicefabrik.io_v1alpha1_dockers.yaml": base64_template('crds/lock.servicefabrik.io_v1alpha1_deploymentlocks.yaml')
```

Comment:
should be dockers.yaml ?

Reply:
The CRDs are generated as yaml file extension.

for yaml file I think both yaml and yml are valid. As it is generated with yaml, would prefer to keep it that way.

Reply:
ok , pls see 1 or 2 of the files are saved as .yml

Reply:
yes, the virtualhost one was yml, changed it accordingly.